app/renderer/ui/theme.scss

```diff
@@ -1,5 +1,7 @@
 @import "app/renderer/ui/colors";
 
+$background-color: $grey-2;
```

Comment:
As discussed, due to the main window also uses the same background color, it should also use it. 
(See [Wallet sketch/design](https://github.com/witnet/sheikah/issues/16)